runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/DokkaMultiModuleFileLayout.kt

```diff
@@ -79,5 +79,27 @@ internal fun DokkaMultiModuleTask.copyChildOutputDirectory(child: AbstractDokkaT
     }
 
     sourceChildOutputDirectory.copyRecursively(targetChildOutputDirectory, overwrite = true)
+    copyNavigationForMultiModule(sourceChildOutputDirectory, outputDirectory.getSafe(), child)
+}
+
+internal fun copyNavigationForMultiModule(
```

Comment:
Hmm. I see. This is the workaround you were talking about! 
I am not quite sure, if we should really go that way 🙄